extensions/eclipse-che-theia-plugin-ext/src/browser/plugin/che-plugin-frontend-service.ts

```diff
@@ -25,14 +25,11 @@ export class ChePluginFrontentService {
     @inject(HostedPluginServer)
     protected readonly hostedPluginServer: HostedPluginServer;
 
-    async getDeployedPlugins(filter: string): Promise<ChePluginMetadata[]> {
-        if (PluginFilter.hasType(filter, '@installed')) {
-            let pluginList = await this.getAllDeployedPlugins();
-            pluginList = PluginFilter.filterPlugins(pluginList, filter);
-            return pluginList;
-        }
-
-        return [];
+    // returns a list of built-in plugins when filter contains '@builtin'
+    async getBuiltInPlugins(filter: string): Promise<ChePluginMetadata[]> {
```

Comment:
Maybe method of the name should be more generic, because it woks not only for `builtin` filter value, I guess?